src/diffusers/pipelines/stable_diffusion/pipeline_onnx_stable_diffusion.py

```diff
@@ -114,17 +114,18 @@ def _encode_prompt(self, prompt, num_images_per_prompt, do_classifier_free_guida
             prompt,
             padding="max_length",
             max_length=self.tokenizer.model_max_length,
+            truncation=True,
             return_tensors="np",
         )
         text_input_ids = text_inputs.input_ids
+        untruncated_ids = self.tokenizer(prompt, padding="max_length", return_tensors="np").input_ids
 
-        if text_input_ids.shape[-1] > self.tokenizer.model_max_length:
+        if not np.equal(text_input_ids, untruncated_ids):
```

Comment:
if not np.equal(text_input_ids.all(), untruncated_ids.all()):

Reply:
Error:
removed_text is not correctly being outputted via logging

Prompt: `cat,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,`

Output:
`The following part of your input was truncated because CLIP can only handle sequences up to 77 tokens: ['']`

Reply:
Good catch! Thanks - this should be fixed :-)

Reply:
Tested changes, verified is it fixed 👍